### ISSUE TITLE
test_compare_to_xls should not be an expected failure

### ISSUE THREAD

**Issue by travs** [openworm/PyOpenWorm#119]
Try AVAL instead of ADAL.

**Comment by travs** [comment]
Getting this result when I run it now:
```
======================================================================
FAIL: Compare the PyOpenWorm connections to the data in the spreadsheet
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/travis/programming/worm/PyOpenWorm/tests/DataIntegrityTest.py", line 275, in test_compare_to_xls
    self.assertEqual(sorted(pow_conns), sorted(xls_conns))
AssertionError: Lists differ: [['ADAL', 'ADAR', 'gapJunction... != [['ADAL', 'ADAR', 'gapJunction...

First differing element 7:
['ADAL', 'AVBR', 'send', '7']
['ADAL', 'AVBR', 'send', '2']

Second list contains 2 additional elements.
First extra element 32:
['PVR', 'ADAL', 'send', '1']

  [['ADAL', 'ADAR', 'gapJunction', '1'],
   ['ADAL', 'ADFL', 'gapJunction', '1'],
   ['ADAL', 'AIBL', 'send', '1'],
   ['ADAL', 'AIBR', 'send', '2'],
   ['ADAL', 'ASHL', 'gapJunction', '1'],
   ['ADAL', 'AVAR', 'send', '2'],
   ['ADAL', 'AVBL', 'send', '4'],
-  ['ADAL', 'AVBR', 'send', '7'],
?                            ^

+  ['ADAL', 'AVBR', 'send', '2'],
?                            ^

+  ['ADAL', 'AVBR', 'send', '5'],
   ['ADAL', 'AVDL', 'send', '1'],
   ['ADAL', 'AVDR', 'gapJunction', '2'],
   ['ADAL', 'AVEL', 'send', '1'],
   ['ADAL', 'AVJR', 'send', '5'],
   ['ADAL', 'FLPR', 'send', '1'],
   ['ADAL', 'PVQL', 'gapJunction', '1'],
+  ['ADAL', 'RICL', 'send', '1'],
   ['ADAL', 'RICL', 'send', '1'],
   ['ADAL', 'RICR', 'send', '1'],
   ['ADAL', 'RIML', 'send', '3'],
   ['ADAL', 'RIPL', 'send', '1'],
   ['ADAL', 'SMDVR', 'send', '2'],
   ['ADAR', 'ADAL', 'gapJunction', '1'],
   ['ADEL', 'ADAL', 'send', '1'],
   ['ADFL', 'ADAL', 'gapJunction', '1'],
   ['ADFL', 'ADAL', 'send', '1'],
   ['AIAL', 'ADAL', 'send', '1'],
   ['ASHL', 'ADAL', 'gapJunction', '1'],
   ['ASHL', 'ADAL', 'send', '1'],
   ['AVDR', 'ADAL', 'gapJunction', '2'],
   ['AWAL', 'ADAL', 'send', '1'],
   ['PVPL', 'ADAL', 'send', '1'],
   ['PVQL', 'ADAL', 'gapJunction', '1'],
   ['PVR', 'ADAL', 'send', '1'],
   ['RMGL', 'ADAL', 'send', '1']]

----------------------------------------------------------------------
Ran 8 tests in 13.182s
```

**Comment by travs** [comment]
This is caused by a naming change between the current database version (which uses things like `AS01`) and an older version serialized as n3 (which uses `AS1`).

These two versions need to be kept in sync.

**Comment by slarson** [comment]
Fixed by #125
